docs/root/api/stats.rst

```diff
@@ -7,7 +7,9 @@ Stats
 ``StatsClient``
 ---------------
 
-To use Envoy Mobile's stats function, obtain an instance of ``StatsClient`` from ``Engine`` (refer to :ref:`api_starting_envoy` for building an engine instance), and store the stats client to create ``Counter`` instances.
+Envoy Mobile's stats function currently supports two type of stats: ``Counter`` and ``Gauge``.
+
+To use Envoy Mobile's stats function, obtain an instance of ``StatsClient`` from ``Engine`` (refer to :ref:`api_starting_envoy` for building an engine instance), and store the stats client to create ``Counter`` or ``Gauge`` instances. The following code examples show how to create a ``Counter``, and it's similar in the way a ``Gauge`` is created.
```

Comment:
```suggestion
To use Envoy Mobile's stats, obtain an instance of ``StatsClient`` from ``Engine`` (refer to :ref:`api_starting_envoy` for building an engine instance), and store the stats client to create ``Counter`` or ``Gauge`` instances:
```
I'd just add an example for gauges since that's what we do everywhere else, and it's nice to be comprehensive.
Also, please cap line widths as we do elsewhere in the file (100 characters IIRC)

Reply:
chatted shortly, keeping it as-is to avoid repetition. will cap the line.